network/src/counters.rs

```diff
@@ -140,17 +168,18 @@ pub static PENDING_NETWORK_REQUESTS: Lazy<IntCounterVec> = Lazy::new(|| {
 /// Counter of pending network events to Health Checker.
 pub static PENDING_HEALTH_CHECKER_NETWORK_EVENTS: Lazy<IntCounterVec> = Lazy::new(|| {
     register_int_counter_vec!(
-        "pending_health_checker_network_events",
-        "Counters(queued,dequeued,dropped) related to pending network notifications to HealthChecker",
+        "libra_network_pending_health_check_events",
```

Comment:
same here... it might be more informative to learn about missed / acked health check events or more specifically when was the last time I request a message and when was the last time I heard from them.

Reply:
I'll come by on the next pass and update the metrics in general.  I just wanted to tie updating the descriptions to the documentation updates.